litmus-portal/frontend/src/views/CreateWorkflow/VerifyCommit/index.tsx

```diff
@@ -350,6 +374,37 @@ const VerifyCommit = forwardRef(
                   />
                 </div>
               </div>
+
+              <div className={classes.summaryDiv}>
+                <div className={classes.innerSumDiv}>
+                  <div style={{ display: 'flex' }}>
```

Comment:
Can you remove this inline styling and mode it to styles folder?

Reply:
@Saranya-jena for single css property this way is better. cc: @amityt